doc/source/whats_new.rst

```diff
@@ -83,6 +83,8 @@ Changelog
 
    - Add reader function for Raw FIF files by `Teon Brooks`_
 
+   - Add example of creating MNE objects from arbitrary data and NEO files by 'Jaakko Leppakangas'_
```

Comment:
I just noticed this as I was rebasing my branch on to master. `Jaakko Leppangas` has the wrong quotes around it which will prevent the doc from building correctly.